### ISSUE TITLE
Load data for report only once using snapshot

### ISSUE THREAD

**Issue by carlobeltrame** [gloggi/tramopoly#5]
Report always takes forever to load when binding all the data. Use [snapshots](https://vuefire.vuejs.org/vuefire/querying.html#one-time-read) instead.

**Comment by carlobeltrame** [comment]
No longer relevant, since scores are calculated on the server now.
